mods/cnc/maps/gdi04c/gdi04c.lua

```diff
@@ -88,7 +88,7 @@ WorldLoaded = function()
 	end)
 
 	nodObjective = nod.AddPrimaryObjective("Destroy all GDI troops.")
-	gdiObjective1 = player.AddPrimaryObjective("Defend the town of Bialystok.")
+	gdiObjective1 = player.AddPrimaryObjective("Defend the town of Białystok.")
```

Comment:
in-mission it displays as Bialystok for me (oddly enough with a normal 'l') - I assumed that char is not a problem since it displays (correctly) in the mission selection screen. maybe a freetype thing?